### PR TITLE
[Improve][Doc] Improve README

### PR DESCRIPTION
**Pull request by caicancai**
## What do these changes do?

1.Distinguish between build and contribute
2.Delete the relevant part of roadmap (reflected in the top issue)

## Related issue number

Closes #xxxx

## Check code requirements

- [ ] tests added / passed (if needed)
- [ ] Ensure all linting tests pass, see [here](https://github.com/alipay/fury/blob/main/CONTRIBUTING.rst) for how to run them

### CODE REVIEW

**Comment by caicancai** [comment]
cc @chaokunyang

File: README.md [hunk 1]
```diff
@@ -364,18 +364,13 @@ Fury provides a class registration option that is enabled by default for such pr
...
-- C++/Rust object graph serialization support.
-- Golang/Rust/NodeJS row format support.
-- ProtoBuffer compatibility support.
-- Protocols for features and knowledge graph serialization.
-- Continuously improve our serialization infrastructure for any new protocols.
-
 ## How to Build
 
-Please read the [CONTRIBUTING](CONTRIBUTING.md) guide for instructions on how to build.
+Please read the [BUILD](BUILDING.md) guide for instructions on how to build.
```
Comment (chaokunyang) [inline README.md:369]: Will renaming to  `DEVELOPMENT` be better? or keep `BUILD`?

Comment (chaokunyang) [reply README.md:369]: Can we move this file to docs/guides? So it will be displayed on website

**Review by chaokunyang — APPROVED** [review]
LGTM

**Pull request merged**